src/rust-crypto/verification.ts

```diff
@@ -653,11 +667,22 @@ export class RustSASVerifier extends BaseRustVerifer<RustSdkCryptoJs.Sas> implem
         if (req) {
             await this.outgoingRequestProcessor.makeOutgoingRequest(req);
         }
+        this.started = true;
```

Comment:
It might be better to call this `accepted`, since the bit we care about is calling `inner.accept()`? It's confusing either way (since it could mean sending either an `m.k.v.start` *or* an `m.k.v.accept`), but I think `accepted` would be slightly less confusing?

(also, it's not the same as `VerificationPhase.Started`)